src/main/java/svc/models/Municipality.java

```diff
@@ -0,0 +1,9 @@
+
+package svc.models;
+
+public class Municipality 
+{
+	public int id;
+	public String municipality;
```

Comment:
I know this variable is all over the place now but it's always a little confusing to have a property the same name as the domain model itself. If it propagates easily, could this be changed to `name` instead of `municipality`?